src/engine/objects/mod.rs

```diff
@@ -648,7 +676,7 @@ where
     GlobalCtx: GlobalContext,
     RequestCtx: RequestContext,
 {
-    pub(crate) fn new(
+    pub fn new(
```

Comment:
Do custom resolvers need to create the directly, rather than going through the ResolverFacade?